specs/Ihec_metadata_specification.md

```diff
@@ -288,133 +333,65 @@ __MRE_PROTOCOL_RESTRICTION_ENZYME__ - The restriction enzyme(s) used in the MRE
 
-<strong>RNA_PREPARATION_5'_RNA_ADAPTER_SEQUENCE</strong> - The sequence of the 5’ RNA adapter used in preparation.
-
-<strong>RNA_PREPARATION_3'_RNA_ADAPTER_SEQUENCE</strong> - The sequence of the 3’ RNA adapter used in preparation.
-
-__RNA_PREPARATION_REVERSE_TRANSCRIPTION_PRIMER_SEQUENCE__ - The sequence of the primer for reverse transcription used in preparation.
-
-<strong>RNA_PREPARATION_5'_DEPHOSPHORYLATION</strong> - The protocol for 5’ dephosphorylation used in preparation.
-
-<strong>RNA_PREPARATION_5'_PHOSPHORYLATION</strong> - The protocol for 5’ phosphorylation used in preparation.
-
-<strong>RNA_PREPARATION_3'_RNA_ADAPTER_LIGATION_PROTOCOL</strong> - The protocol for 3’ adapter ligation used in preparation.
-
-<strong>RNA_PREPARATION_5'_RNA_ADAPTER_LIGATION_PROTOCOL</strong> - The protocol for 5’ adapter ligation used in preparation.
-
-__LIBRARY_GENERATION_PCR_TEMPLATE_CONC__ - The PCR template concentration for library generation.
-
-__LIBRARY_GENERATION_PCR_POLYMERASE_TYPE__ - The PCR polymerase used for library generation
-
-__LIBRARY_GENERATION_PCR_THERMOCYCLING_PROGRAM__ - The thermocycling program used for library generation.
-
-__LIBRARY_GENERATION_PCR_NUMBER_CYCLES__ - The number of PCR cycles used for library generation.
-
-__LIBRARY_GENERATION_PCR_F_PRIMER_SEQUENCE__ - The sequence of the PCR forward primer used for library generation.
-
-__LIBRARY_GENERATION_PCR_R_PRIMER_SEQUENCE__ - The sequence of the PCR reverse primer used for library generation.
-
-__LIBRARY_GENERATION_PCR_PRIMER_CONC__ - The concentration of the PCR primers used for library generation.
-
-__LIBRARY_GENERATION_PCR_PRODUCT_ISOLATION_PROTOCOL__ - The protocol for isolating PCR products used for library generation.
-
-__TEMPLATE_TYPE__ - mRNA or cDNA - The type of template.
-
-__AMPLIFIED__ - True or False - Is the sample amplified?
-
-__PREPARATION_INITIAL_MRNA_QNTY__ -The initial mRNA quantity used in preparation.
-
-__PREPARATION_REVERSE_TRANSCRIPTION_PROTOCOL__ - The protocol for reverse transcription used in preparation.
-
-__PREPARATION_PCR_NUMBER_CYCLES__ - The number of PCR cycles used to amplify.
-
-__LIBRARY_GENERATION_PROTOCOL__ - The protocol used to generate the library.
-
-__LIBRARY_GENERATION_FRAGMENTATION__ - The fragmentation method used in the library protocol.
-
-__LIBRARY_GENERATION_FRAGMENT_SIZE_RANGE__ - The fragment size range of the preparation.
-
-<strong>LIBRARY_GENERATION_3'_ADAPTER_SEQUENCE</strong> - The sequence of the 3' adapter used for library generation.
-
-<strong>LIBRARY_GENERATION_5'_ADAPTER_SEQUENCE</strong> - The sequence of the 5' adapter used for library generation.
-
-### total-RNA-seq
-
-__EXPERIMENT_TYPE:__ total-RNA-Seq
-
-__EXPERIMENT_ONTOLOGY_URI__ - (Ontology) links to experiment ontology information.
+__MOLECULE:__ (Controlled Vocabulary) 'polyA RNA', 'total RNA', 'nuclear RNA' or 'cytoplasmic RNA'.
```

Comment:
Again the molecule here needs to be revised for smRNA-Seq

Reply:
Added 'small RNA' to molecule controlled vocabulary.